doc/source/whatsnew/v0.24.0.txt

```diff
@@ -829,4 +829,5 @@ Other
 - :meth:`DataFrame.nlargest` and :meth:`DataFrame.nsmallest` now returns the correct n values when keep != 'all' also when tied on the first columns (:issue:`22752`)
 - :meth:`~pandas.io.formats.style.Styler.bar` now also supports tablewise application (in addition to rowwise and columnwise) with ``axis=None`` and setting clipping range with ``vmin`` and ``vmax`` (:issue:`21548` and :issue:`21526`). ``NaN`` values are also handled properly.
 - Logical operations ``&, |, ^`` between :class:`Series` and :class:`Index` will no longer raise ``ValueError`` (:issue:`22092`)
--
+- Added specific exception classes to fix bare except statements in `pandas/core` (:issue:``22873)
```

Comment:
Thanks for adding this, but not sure if it's worth adding this change to the release note. It's more of a internal change that should not have any impact on users.